kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/Configuration.java

```diff
@@ -49,10 +50,14 @@
 
 /**
  * The root of the proxy configuration.
+ * <br>
+ * <br>
+ * Note that {@code adminHttp} is accepted as an alias for {@code management}.  Use of {@code adminHttp} is deprecated since 0.11.0
+ * and will be removed in a future release.
  */
-@JsonPropertyOrder({ "adminHttp", "filterDefinitions", "defaultFilters", "virtualClusters", "filters", "micrometer", "useIoUring", "development" })
+@JsonPropertyOrder({ "management", "filterDefinitions", "defaultFilters", "virtualClusters", "filters", "micrometer", "useIoUring", "development" })
 public record Configuration(
-                            @Nullable AdminHttpConfiguration adminHttp,
+                            @Nullable @JsonAlias("adminHttp") ManagementConfiguration management,
```

Comment:
In other cases we didn't use `@JsonAlias` (even though it's the simplest way to do it) so that we could log a warning about use of the old name.

Reply:
Fixed.  I used a StdDeserializer to log the deprecation warning which is much less intrusive than a constructor changes.